packages/react-ui-core/src/Dropdown/DropdownMenu.js

```diff
@@ -90,29 +93,25 @@ export default class DropdownMenu extends Component {
 
   render() {
     const {
-      className,
       options,
       theme,
       handleSelectionHover,
+      className,
       onSelection,
       ...props
     } = this.props
 
-    const { activeIndex } = this.state
-
     return (
       <List
-        className={cn(
+        role="button"
+        className={classnames(
           className,
-          theme.List,
           theme.DropdownMenu
         )}
         items={options}
-        Item={ItemWrapper}
-        nodeType="div"
-        selectedIndex={activeIndex}
-        handleSelectionClick={this.handleSelection}
-        handleSelectionHover={this.activateOption}
+        activeIndex={this.state.activeIndex}
+        onClick={this.handleSelection}
+        onMouseEnter={this.activateOption}
```

Comment:
What does activateOption mean? Is that highlighting an item? If so that could be clearer...

Reply:
It really could mean anything I think, that's why it's vague. It generally would mean hovered or selected with up / down keyboard stroke, but it's not _technically_ selected yet. Make sense yet? ;)

Reply:
thats why I said highlighting because that seems to make it more clear while leaving any opening for it to be a little vague.

Reply:
Ok, I can change it.